systems/summit_user_guide.rst

```diff
@@ -3269,20 +3307,23 @@ user environment ``PAMI_PMIX_USE_OLD_MAPCACHE=1`` and
 CUDA 10.1 Known Issues
 ----------------------
 
-Default nvprof setting clobbers ``LD_PRELOAD``, interfering with SpectrumMPI
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Intermittent failures with \`nvprof\` (Identified: July 11, 2019)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
section underlines must be at least as long as the text above them

Reply:
fixed